mkdocs/docs/HPC/FAQ.md

```diff
@@ -1,6 +1,227 @@
+* Replace the "`-placeholder text-`" with real entries. This notation is used to ensure `qsub` rejects invalid options.
+* To use a situational parameter, remove 1 '`#`' at the beginning of the line.
+
+```shell
+#!/bin/bash
+
+# Basic parameters
+#PBS -N jobname           ## Job name
+#PBS -l nodes=1:ppn=2     ## 1 node, 2 processors per node (ppn=all to get a full node)
+#PBS -l walltime=01:00:00 ## Max time your job will run (no more than 72:00:00)
+
+# Situational parameters: remove one '#' at the front to use
+##PBS -l gpus=1            ## GPU amount (only on accelgor or joltik)
+##PBS -l mem=32gb          ## If not used, memory will be available proportional to the max amount
+##PBS -m abe               ## Email notifications (abe=aborted, begin and end)
+##PBS -M -email_address-   ## ONLY if you want to use a different email than your VSC address
+##PBS -A -project-         ## Project name when credits are required (only Tier 1)
+
+##PBS -o -filename-        ## Output log
+##PBS -e -filename-        ## Error log
+
+
+module load [module]
+module load [module]
+
+cd $PBS_O_WORKDIR         # Change working directory to the location where the job was submmitted
+
+[commands]
+```
+
+Some [Job script examples](./jobscripts_examples).
+
+## Troubleshooting jobs
+
+### My modules don't work together
+
+When incompatible modules are loaded, you might encounter an error like this:
+
+Lmod has detected the following error: A different version of the 'GCC' module
+is already loaded (see output of 'ml').
+You should load another 'foss' module for that is compatible with the currently
+loaded version of 'GCC'.
+Use 'ml spider foss' to get an overview of the available versions.
+
+Modules from the same toolchain always work together, and modules from a
+_different version of the same toolchain_ never work together.
+
+An overview of compatible toolchains can be found here:
+<https://docs.easybuild.io/en/latest/Common-toolchains.html#overview-of-common-toolchains>.
+
+See also: How do I choose the job modules?
+
+### My job takes longer than 72 hours
+
+The 72 hour walltime limit will not be extended. However, you can work around this barrier:
+
+* Check that all available resources are being used. See also:
+    * How many cores/nodes should I request?
+    * My job is slow
+    * My job isn't using any GPUs
+* Use a faster [cluster](https://www.ugent.be/hpc/en/infrastructure)
+* Divide the job into more parallel processes
+* Divide the job into shorter processes, which you can submit as separate jobs
+* Use the built-in checkpointing of your software
+
+### Job failed: SEGV Segmentation fault
+
+Any error mentioning SEGV or Segmentation fault/violation has something to do with a memory error.
+If you weren't messing around with memory-unsafe applications or programming, your job probably hit its memory limit.
```

Comment:
memory limits don't trigger segfaults. those give OOM errors i think

Reply:
@stdweird I did a copy of the current pr, should we remove this specific section?